### DISCUSSION TITLE
It's time to cleanup the Experiments list in Elementor Settings

### DISCUSSION THREAD

**Discussion by d19dotca** [elementor/elementor#15463] [Feature Request]
The Experiments list in Elementor Settings as of Elementor v3.4.0-dev5 & Elementor Pro v3.4.0-dev21 has now grown to a pretty lengthy list, and hasn't been cleaned up since Experiments were started (I think back in 3.1). This raises several questions and concerns: 

1. There are too many experiments for users to keep track of - there are currently 11 as of the current developer edition.
2. A few of the experiments were presented as full features such as Video Playlist, Form Submissions, Landing Pages, and more. Why are these listed as Experiments at all if they were marketed as a fully functioning feature? Shouldn't those have been removed from Experiments and fully productized?
3. Something fairly simple like the "Admin Bar"... I'm a bit curious why something like that would even be an experiment in the first place. I'm assuming there's justification for it, but I can't think of anything that would explain that one. 
4. Since Experiments were introduced, I don't think I've seen a single experiment actually "graduate" to being fully productized yet in all this time, which seems strange.

**What problem is your feature request going to solve? Please describe.**
Cleaning up the Experiments list in order to improve the user experience, especially for beginners. This will also progress the Elementor product.

**Describe the solution you'd like**
I'd like the Elementor team to review the Experiments list, determine what is truly "unstable" enough to still need to be an Experiment, and make the necessary changes to clean up the list of experiments by promoting the stable ones to a proper productized release. 

I'd also like for new features being introduced in new versions which are actually experiments to be very clear that they're experiments only and thus unstable and not recommended for production sites, etc. In my opinion the marketing materials such as for the most recent Video Playlist widget promote the widget as if it was a fully functional and stable version, when by definition of it being an experiment it must not be too stable yet for a production level website. These need to be cleaned up and clarified, in my opinion.

**Additional context**
I love the idea of Experiments, but I (and others based on comments in other posts and on the Facebook group) feel the Experiments feature is perhaps being a little bit "abused" now. Some experiments make sense that they'd take a while before being fully productized (such as the Optimize DOM, Improve Asset Loading, etc experiments because of the high risk with all the customizations users make), but many experiments also seem to be "stuck" as an experiment despite seeming totally stable. Also if any experiments are being defaulted to "on" for new sites (as several of them do), then I'd assume this means they're stable enough to be production-worthy? In such a case, may be worth removing it as an experiment and making it fully productized (but maybe put into a "legacy compatibility" mode for those types of experiments so they can be disabled still). I hope that the Elementor team will find the above useful and make efforts to clean up the Experiments list and improve the documentation / marketing around new features which are actually just experiments still.

**Comment by fabigrafo** [comment]
Jup, +1!

By the way, they are working on it, maybe: https://github.com/elementor/hello-theme/issues/150#issuecomment-857907229

**Comment by d19dotca** [comment]
> Jup, +1!
> 
> By the way, they are working on it, maybe: https://github.com/elementor/hello-theme/issues/150#issuecomment-857907229

Yes, that was actually one of the latest things that prompted me to make this request official. Hopefully they make good on that. 😀

**Comment by arielk** [comment]
Hi, we are going to merge stable experiments into the product.

You can see in the next version 3.10 some experiments merged into the core: https://github.com/elementor/elementor/pull/20462/commits

Thanks for your feedback!

**Reply by fabigrafo** [reply]
Sounds good! 😁
